components/AirdropLogin.vue

```diff
@@ -108,22 +109,46 @@
   </div>
 </template>
 <script lang="ts">
-import { Vue, Component, Prop } from 'vue-property-decorator'
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator'
 import { CONNECT_WALLET_TYPES } from '~/constant'
 
+export enum CosmosArrdessFormat {
+  head = 'cosmos1',
+  length = 45
+}
+export enum OsmoArrdessFormat {
+  head = 'osmo1',
+  length = 43
+}
+
```

Comment:
I think using regex is cleaner, and can filter invalid characters at the same time